other/blacklistipchecker.py

```diff
@@ -45,10 +44,10 @@ def test_ip(ip: str) -> tuple[str, list] | tuple[str, str] | None:
             IP(x)
         except Exception as ex:
             print("Argument isn`t a valid ip(" + x + ")\n")
-            print(ex + "\n Skipping argument")
+            print(str(ex) + "\n Skipping argument")
```

Comment:
Python f-strings are faster...
```suggestion
            print(f"{ex}\n Skipping argument")
```